package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-claim",
-  "version": "0.20.0",
+  "version": "0.20.1",
```

Comment:
Given the size of change here and that it isn't a bug, I'd bump this up to 0.21.0

Reply:
bumped